README.md

```diff
@@ -14,6 +14,9 @@ exports.config = {
     // Output json and html will go in this folder.
     outdir: 'timelines',
 
+    // Set the name of the html file. Defaults to index.html.
+    htmlFile: 'results.html'
```

Comment:
I imagine that the json file name would eventually be configurable as well. But since it was slightly more than just adding the configuration (you would have to replace the reference in the html file) and I didn't have a specific need for that, I didn't added that here. But with `outputFileName` being generic, it doesn't say which of the two files it is naming. I suppose it could be used to affect both files, naming them the same just with different extensions.